README.md

```diff
@@ -214,10 +214,15 @@ The PIConGPU Team expresses its thanks to:
 - Stefan Tietze
 - Benjamin Worpitz
 - Erik Zenker
+- Alexander Matthes
```

Comment:
I like, where my name is: At the end of a **long** list of other people, which can do stuff instead of me. :>